File: solutions/026.remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if not nums:
            return 0
        
        pre = nums[0]
        length = 1
        for i, num in enumerate(nums):
            if num == pre:
                continue
            else:
                nums[length] = num
                pre = num
                length += 1
                
        return length
        

File: solutions/026.remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        if (nums == null || nums.length == 0){
            return 0;
        }
        int pre = 1;
        for (int i = 1; i < nums.length; i++){
            if(nums[i] != nums[i-1]){
                nums[pre++] = nums[i];
               
            }
        }
        return pre;
    }
}